features/place/duplicate.feature

```diff
@@ -12,7 +12,7 @@ Feature: Test creating places
     Given I create a minimal place and save the "id" as "originalPlaceId" then I should get a "201" response code
     When I publish the place at "/places/%{originalPlaceId}"
     And I reject the place at "/places/%{originalPlaceId}" with reason "Rejected"
-    Then I wait for the place with url "/places/%{originalPlaceId}" to be indexed
```

Comment:
I changed this because the `I wait * to be indexed` only checks if it's on sapi, not if it already has been `REJECTED`. So If you have a fast way to run your tests(e.g., `OrbStack` this test will fail.